workflow/rules/common.smk

```diff
@@ -56,30 +47,38 @@ wildcard_constraints:
     type="N|T|R",
 
 
-if config["trimmer_software"] == "fastp_pe":
+if config.get("trimmer_software", "None") == "fastp_pe":
     alignment_input = lambda wilcards: [
         "prealignment/fastp_pe/{sample}_{flowcell}_{lane}_{type}_fastq1.fastq.gz",
         "prealignment/fastp_pe/{sample}_{flowcell}_{lane}_{type}_fastq2.fastq.gz",
     ]
-elif config["trimmer_software"] == "None":
+elif config.get("trimmer_software", "None") == "None":
     alignment_input = lambda wildcards: [
         get_fastq_file(units, wildcards, "fastq1"),
         get_fastq_file(units, wildcards, "fastq2"),
     ]
 
-if config.get("alignment_software", None) == "gpu":
-    # extract_reads_input = {"bam": "alignment/apply_bqsr_gpu/{sample}_{type}.bqsr.dedup.bam", "bai": "alignment/apply_bqsr_gpu/{sample}_{type}.bqsr.dedup.bam"}
-    extract_reads_input = "alignment/apply_bqsr_gpu/{sample}_{type}.bqsr.dedup.bam"
-    extract_reads_input_bai = "alignment/apply_bqsr_gpu/{sample}_{type}.bqsr.dedup.bam.bai"
-else:
-    # extract_reads_input = {"bam": "alignment/bwa_mem/{sample}_{type}.bam", "bai": "alignment/bwa_mem/{sample}_{type}.bam.bai"}
-    extract_reads_input = "alignment/bwa_mem/{sample}_{type}.bam"
-    extract_reads_input_bai = "alignment/bwa_mem/{sample}_{type}.bam.bai"
+
+if config.get("alignment_software", "None") == "gpu":
```

Comment:
Seems like this part is not necessary anymore or am I missing something?

Reply:
true

Reply:
let's remove the option of setting alignment software until we add more aligners? @marrip

Reply:
Might be good to keep things simple. And anything parabricks-related is not going to be included here anyways.